ch_pipeline/core/telescope.py

```diff
@@ -87,6 +93,7 @@ class CHIME(telescope.PolarisedTelescope):
     layout = config.Property(default=None)
```

Comment:
What would happen if layout is not provided?  Should we have a default value here?

Reply:
Yeah, good point; this is something we may as well settle on now. 

`_load_layout` will raise an exception if `layout` is not already set, but in `_finalise_config`, the call to `_load_layout` is conditioned on whether `layout` has been set; so, it seems that you are just (silently) left with an instance with no feed information in this case, which I imagine will tend to result in a crash somewhere downstream anyway. So, we should probably either crash here to require the user to be explicit in setting the layout (which would also be consistent usage with requiring a `layout` argument when building an instance  with `CHIME.from_layout`), or for convenience, set a default. In the latter case, do we have a reason to not use the most recent layout as the default?

Reply:
Yeah I think I would probably issue a warning that the parameter has not been set and then default to the most recent layout available in the files.